recipes/f90nml/bld.bat

```diff
@@ -0,0 +1,2 @@
+"%PYTHON%" setup.py install 
+if errorlevel 1 exit 1
```

Comment:
Please drop both `bld.bat` and `build.sh` in favor of the `script` line. (See the example recipe.)